src/main/java/frc/robot/Subsystems/IntakeSubsystem.java

```diff
@@ -51,47 +53,53 @@ public double getWAxisSpeedMultiplier(double axisValue) {
     }
 
     public void releaseGamePiece(double speedMultiplier) {
-        if (state == IntakeGamePiece.CONE) {
+        if (state == GamePiece.CONE) {
             releaseCone(speedMultiplier);
-        } else if (state == IntakeGamePiece.CUBE) {
+        } else if (state == GamePiece.CUBE) {
             releaseCube(speedMultiplier);
         }
     }
 
     public void intakeGamePiece(double speedMultiplier) {
-        if (state == IntakeGamePiece.CONE) {
+        if (state == GamePiece.CONE) {
             pickUpCone(speedMultiplier);
-        } else if (state == IntakeGamePiece.CUBE) {
+        } else if (state == GamePiece.CUBE) {
             pickUpCube(speedMultiplier);
         }
     }
 
     public void pickUpCone(double speedMultiplier) {
-        // TODO: tune motor speeds
         intakeMotorInner.set(-0.4);
```

Comment:
I thought we were tuning these to run at different speeds?

Reply:
they are fine for now, i'll add one singular comment at the top of the class